### ISSUE TITLE
Missing hover state on a button in the 18004 blog

### ISSUE THREAD

**Issue by steven1046** [OpenLiberty/blogs#13]
The stack overflow button in the following blog has no hover state:
https://openliberty.io/blog/2018/12/14/microprofile21-18004.html

**Comment by lauracowen** [comment]
What syntax should be used to give hover state? I can't see how to do alt text in Asciidoc - the Asciidoctor user guide gives example of adding a title but the title is displayed below the image, not as hover text.

**Comment by steven1046** [comment]
Nevermind, when you hover over the button image the cursor changes into a hand which I think is enough. We can close this.
